platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
I'm personally indifferent to adding a new dependency that passes OWASP and is scoped only for testing. That is a very low-risk operation.